### ISSUE TITLE
Windows - Crossbar 18.5.1 - Python 3.6.5 (x86) - Failed to start native worker

### ISSUE THREAD

**Issue by Tsunami2069** [crossbario/crossbar#1329]
Hello.

After upgrading to the latest crossbar (18.5.1), I got an error to which Mr. @oberstet so kindly informed me that Crossbar was no longer supporting Python 2.

I've since then Installed Python 3.6.5 and installed crossbar.
However, my crossbar still won't start.

This time, i'm getting an error starting the native worker.

Here is a full log in debug level:
[Log.txt](https://github.com/crossbario/crossbar/files/2014432/Log.txt)

I might add that this router and it's configuration was working perfectly with Python 2 and Crossbar 18.4.1.

Can any1 please help?
Thank you.

**Comment by om26er** [comment]
Confirmed, I was able to reproduce that bug as well, (sorry took a while, had to install Build Tools there).

**Comment by Tsunami2069** [comment]
@om26er 
No problem (about taking a while).
We can keep developing on our old environment (Python 2 and Crossbar 18.4.1) for now.

Thank you very much for your help.
I'll keep a look out on this issue for further updates.

**Comment by om26er** [comment]
@Tsunami2069 The issue is fixed in Crossbar 18.5.2 released last night. Please update and report back.

**Comment by Tsunami2069** [comment]
@om26er 
Hehe, unfortunately, I can't test personally because, last night, upon some research into this and similar topics, I found out something quite interesting, which is that Windows can now run a native Linux distro on it as a subsystem.
Apparently, this feature was released on the Anniversary Update of Windows and I never heard of it.
I installed Linux on my Windows machine, installed Crossbar to the latest version (18.5.1) and it ran beautifully.

However, my partner still has his Windows & Python 3 & Crossbar installation (that was giving out the same error). I asked him to update and test and YES, it's working perfectly again.

Thank you very much for your help.
